tests/Tests/600__ai_apps/620__intel_aikit/620__intel_aikit.robot

```diff
@@ -27,7 +27,7 @@ Verify intel aikit Is Available In RHODS Dashboard Explore Page
 
 Verify Inetl AIKIT Operator Can Be Installed Using OpenShift Console
    [Tags]   Tier2
-   ...      ODS-760
+   ...      ODS-760     ODS-1237
```

Comment:
could u plz maintain the same format as in 
https://github.com/red-hat-data-services/ods-ci/pull/292/files

Reply:
Yes I've committed the changes